practice.css

```diff
@@ -150,6 +150,11 @@ h2.subtitle {
   border-radius: 10px;
   background-color: antiquewhite;
   text-align: center;
+  overflow: scroll;
```

Comment:
Don't add overflow on `.history` class instead use `overflow` on `.history-table` class to only scroll the table not the entire container.